### PR TITLE
gpu-aml: simplify build. strip

### PR DESCRIPTION
**Pull request by lrusak**
thanks @stefansaraev 

@codesnake ok with you?

### CODE REVIEW

File: packages/linux-drivers/gpu-aml/package.mk [hunk 1]
```diff
@@ -34,15 +34,12 @@ PKG_IS_ADDON="no"
 PKG_AUTORECONF="no"
 
 make_target() {
-  cd mali
-  LDFLAGS="" make V=1 \
-    -C $(kernel_path) M=$ROOT/$PKG_BUILD/mali \
-    ARCH=$TARGET_KERNEL_ARCH \
-    CROSS_COMPILE=$TARGET_PREFIX \
+  LDFLAGS="" make -C $(kernel_path) M=$PKG_BUILD/mali \
```
Comment (stefansaraev) [inline packages/linux-drivers/gpu-aml/package.mk]: $ROOT/$PKG_BUILD

File: packages/linux-drivers/gpu-aml/package.mk [hunk 2]
```diff
@@ -34,15 +34,12 @@ PKG_IS_ADDON="no"
...
     CONFIG_MALI400=m CONFIG_MALI450=m
 }
 
 makeinstall_target() {
-  mkdir -p $INSTALL/lib/modules/$(get_module_dir)/mali
-  cp *.ko $INSTALL/lib/modules/$(get_module_dir)/mali
+  LDFLAGS="" make -C $(kernel_path) M=$PKG_BUILD/mali \
```
Comment (stefansaraev) [inline packages/linux-drivers/gpu-aml/package.mk]: $ROOT/$PKG_BUILD

**Comment by codesnake** [comment]
@lrusak Sure is OK with me, but please check @stefansaraev notes.

**Comment by lrusak** [comment]
Updated, thanks.

**Comment by codesnake** [comment]
:+1:

**Comment by lrusak** [comment]
merge please

**Pull request merged**